be.telenet.user.qsstv.yml

```diff
@@ -0,0 +1,79 @@
+app-id: be.telenet.user.qsstv
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qsstv
+rename-desktop-file: qsstv.desktop
+rename-icon: qsstv
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
```

Comment:
I have tried replacing `--filesystem=home` with `--persist=qsstv`, however the `qsstv` directory was not created inside the sandbox for some reason and also there were some other issues like saved images/templates being created with a `.xdp- prefix` for some reason (maybe a portal bug?).

So, `--filesystem=home` is good for now.

Reply:
qsl